README.md

```diff
@@ -29,26 +32,27 @@
 - [Overview / Resources](#overview--resources)
 - [Get Started](#get-started)
 - [Configure](#configure)
-    * [Polyfills required](#polyfills-required)
```

Comment:
We need to use 4 space indentation here for the docs on imgix.com to render correctly.